test/utils/index.js

```diff
@@ -76,5 +76,23 @@ describe('our index utilities', () => {
       expect(splitIntoRows([9, 8, 7, 6, 5, 4, 3], 2)).to.deep.equal([[9, 8], [7, 6], [5, 4], [3]]);
     });
   });
+
+  describe('have a abbrNum utility that', () => {
+    it('returns abbreviated a rounded number based on (1000(n) & maximum trillion)', () => {
+      var sourceNumberSet = [123,1266,125468,77547959,388475766449,48242478968789,482424789687893242];
+      var formattedNumberSet = [123,'1.3k','125.47k','77.548m','388.476b','48.242t','482,424.79t'];
+      for (let i = 0; i < sourceNumberSet.length;i++) {
+        expect(abbrNum(sourceNumberSet[i],i)).to.be.equal(formattedNumberSet[i]);
+      }
+    });
+
+    it('returns no abbreviation if a negative number or 0', () => {
+      var sourceNumberSet = [-12341235,0];
+      var formattedNumberSet = [-12341235,0];
+      for (let i = 0; i < sourceNumberSet.length;i++) {
+        expect(abbrNum(sourceNumberSet[i],i)).to.be.equal(formattedNumberSet[i]);
+      }
+    });
+  });
 });
```

Comment:
There are 23 linting errors introduced by this PR. Can you please run `npm run lint` and fix the errors?

Reply:
Fixed lint errors.

See Commit: 20c49043f37c6a2aaf68864cb693d330d15046ac